model_file_cb_equiv.txt

```diff
@@ -0,0 +1,16 @@
+Version 8.11.0
```

Comment:
[Minor] Did you intend to include these .txt files? If not, consider removing them from the PR.

Reply:
Yes I had not intention for .txt files to be include. Gonna remove them soon